src/cljs/athens/views.cljs

```diff
@@ -52,14 +52,6 @@
       (dispatch [:alert/unset]))))
 
 
-(defn file-cb
```

Comment:
I moved this component to `app_toolbar` since I did a file search and this piece of code is not used elsewhere so this would not affect functionality.